chrome/content/zotero/xpcom/utilities.js

```diff
@@ -1527,14 +1484,14 @@ Zotero.Utilities = {
 	 */
 	"itemToCSLJSON":function(zoteroItem) {
 		if (zoteroItem instanceof Zotero.Item) {
-			zoteroItem = zoteroItem.toArray();
+			zoteroItem = Zotero.Utilities.Internal.itemToExportFormat(zoteroItem);
```

Comment:
In CSL, what is "entry" type? Would that be a good candidate for generic items, like note, attachment, and document? We can continue mapping these to "article", I don't particularly mind, but it should be made explicit.